File: app/src/main/java/id/poncoe/notificationrealtime/notifikasi/java/Notifikasi.java
package id.poncoe.notificationrealtime.notifikasi.java;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;
import android.view.View;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import android.widget.Button;
import android.widget.TextView;

import id.poncoe.notificationrealtime.*;
import id.poncoe.notificationrealtime.MainActivity;

/**
 * Created by Poncoe on 26/12/16.
 */

public class Notifikasi extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.notifikasi);

        if (getSupportActionBar() != null)
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        String openURL = getIntent().getStringExtra("openURL");

        final TextView textView = (TextView)findViewById(R.id.debug_view);
        textView.setText("URL from additionalData: " + openURL);


        Button onBackButton = (Button)(findViewById(R.id.back_button));
        onBackButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getApplicationContext(), MainActivity.class);
                startActivity(intent);
            }
        });

        WebView webView = (WebView)(findViewById(R.id.webview));
        webView.getSettings().setJavaScriptEnabled(true);
        webView.setWebViewClient(new WebViewClient());
        if (openURL == null) {
            webView.loadUrl("https://github.com/poncoe");
        } else {
            webView.loadUrl(openURL);
        }


    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {

        if (item.getItemId() == android.R.id.home)
            startActivity(new Intent(Notifikasi.this, MainActivity.class));
            finish();

        return super.onOptionsItemSelected(item);
    }

    @Override
    public void onBackPressed() {
    }
}

File: app/src/main/java/id/poncoe/notificationrealtime/notifikasi/kotlin/Notifikasi.kt
package id.poncoe.notificationrealtime.notifikasi.kotlin

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.MenuItem
import android.view.View
import android.webkit.WebView
import android.webkit.WebViewClient
import android.widget.Button
import android.widget.TextView

import id.poncoe.notificationrealtime.MainActivity
import id.poncoe.notificationrealtime.R

/**
 * Created by Poncoe on 26/12/16.
 */

class Notifikasi : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.notifikasi)

        if (supportActionBar != null)
            supportActionBar!!.setDisplayHomeAsUpEnabled(true)

        val openURL = intent.getStringExtra("openURL")

        val textView = findViewById<View>(R.id.debug_view) as TextView
        textView.text = "URL from additionalData: " + openURL!!


        val onBackButton = findViewById<View>(R.id.back_button) as Button
        onBackButton.setOnClickListener {
            val intent = Intent(applicationContext, MainActivity::class.java)
            startActivity(intent)
        }

        val webView = findViewById<View>(R.id.webview) as WebView
        webView.settings.javaScriptEnabled = true
        webView.webViewClient = WebViewClient()
        if (openURL == null) {
            webView.loadUrl("https://github.com/poncoe")
        } else {
            webView.loadUrl(openURL)
        }


    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {

        if (item.itemId == android.R.id.home)
            startActivity(Intent(this@Notifikasi, MainActivity::class.java))
        finish()

        return super.onOptionsItemSelected(item)
    }

    override fun onBackPressed() {}
}